.travis.yml

```diff
@@ -31,6 +31,10 @@ group: travis_latest
 before_install:
   - |
 
+      # Update to the latest yarn as older versions have known issues
```

Comment:
To keep the test time low in certain cases where `yarn` is not used, let's move this under `install`.

Reply:
@antross I'll make the change before merging.